devito/builtins.py

```diff
@@ -255,6 +255,8 @@ def initialize_function(function, data, nbl, mapper=None, mode='constant',
         accepted.
     name : str, optional
         The name assigned to the operator.
+    pad_halo : bool, optional
+        Whether to also pad the finite-difference halo.
```

Comment:
Drop "finite-difference" or replace with something else

Reply:
But that's the FD halo from the space order, why droping it?

Reply:
the halo is a concept that applies to any (un)structured grid computation, so it's broader than FD

Reply:
Changed to outer halo